doc/Security.xml

```diff
@@ -947,13 +987,290 @@ Added certificate-based client authentication</revremark>
       <para>This specification defines a set of commands to configure and manage a device’s IEEE 802.1X configurations, both for wireless and hardwired network interfaces.  It assumes that IEEE 802.1X configuration and reconfiguration is performed outside of the IEEE 802.1X-secured network.</para>
       <para>Many schema elements in this specification include Dot1X as shorthand for IEEE 802.1X.  This convention increases the readability of source code generated from the WSDL.</para>
     </section>
+    <section xml:id="section_b4m_rs4_rwb">
+      <title xml:id="title_mjt_nzq_swb">Authorization Servers</title>
+      <para>This section describes use of external authorization servers that can be used for
+        providing authentication of users and authorizing access to both device services and
+        external services, like cloud storage.</para>
+      <para>OAuth 2.0 is a specification that describes how to issue access tokens. It is defined in [RFC6749]. OpenID Connect has been developed as an extension of OAuth 2.0.</para>
+      <para>OAuth 2.0 includes the definition of a web API called "authorization endpoint". Requests to this API must include a parameter called <emphasis>response_type</emphasis> which can be either <emphasis>code</emphasis> or <emphasis>token</emphasis>. OpenID Connect extends the OAuth 2.0 specification, by adding the values <emphasis>id_token</emphasis> and <emphasis>none</emphasis>for <emphasis>response_type</emphasis> and allowing any combination of <emphasis>code</emphasis>, <emphasis>token</emphasis> and <emphasis>id_token</emphasis>.</para>
+      <para>Any request for ID tokens must include <emphasis>openid</emphasis> in the <emphasis>scope</emphasis> request parameter.</para>
+      <section xml:id="section_acl_4zq_swb">
+        <title>User authentication and authorization</title>
+        <para>Users can retrieve JWTs from an OpenID Connect server by authenticating and authorizing access to resources by following the OAuth2 Authorization Code Flow.</para>
+        <para>The Authorization Code Flow is summarized in <xref linkend="_Ref350769588"/>.</para>
+        <figure xml:id="_Ref350769588">
+          <title>OpenID Connect authorization code flow for users</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="media/Security/image6.svg" contentwidth="159.97mm"/>
+            </imageobject>
+          </mediaobject>
+        </figure>
+        <para>A device can be setup to use JWTs obtained with this flow to allow access to its internal web interface for externally authenticated users. In that case an authorization server configuration with type OAuthAuthorizationCode should be created.</para>
+        <para>Access to the camera can be granted to users bearing either an ID Token or an Access Token, as long as the claims specified in <xref linkend="_Ref395181339"/> are present.</para>
```

Comment:
Typically, an access token is not used for authentication at the device, only for authentication at other services. The device should not interpret access tokens.

Reply:
I had a discussion with @svefredrik about this. Initially we were stating that the device should validate identity tokens, eventually we changed the text stating that the camera can validate any token, as long as it is correctly singed and has the mandatory claims. No difference between id token, access tokens or whatever.